sources/packages/api/src/route-controllers/student/student.students.controller.e2e-spec.ts

```diff
@@ -81,7 +79,6 @@ describe("Test ATBC Controller", () => {
         .expect(HttpStatus.OK);
     } finally {
       await studentService.remove(fakeStudent);
-      await userService.remove(simsUser);
```

Comment:
I had to delete this line of code because it is causing an error that the delete cascade rule would have prevented. We may now have to clean up manually.